doc/neps/roadmap.rst

```diff
@@ -0,0 +1,116 @@
+=============
+NumPy Roadmap
+=============
+
+This is a live snapshot of tasks and features we will be investing resources
+in. It may be used to encourage and inspire developers and to search for
+funding.
+
+Interoperability protocols & duck typing
+========================================
```

Comment:
this heading should be single underline, and the ``__array_function__`` and ``Array Duck-Typing`` ones should be bullet points instead.